cmdstanpy/model.py

```diff
@@ -169,9 +169,19 @@ def __init__(
             if not cmdstan_version_before(
                 2, 27
             ):  # unknown end of version range
-                model_info = self.src_info()
-                if 'parameters' in model_info:
-                    self._fixed_param |= len(model_info['parameters']) == 0
+                try:
+                    model_info = self.src_info()
+                    if 'parameters' in model_info:
+                        self._fixed_param |= len(model_info['parameters']) == 0
+                except (
+                    ValueError,
+                    RuntimeError,
+                    OSError,
+                    subprocess.CalledProcessError,
```

Comment:
I think ValueError is the only thing that should be raised here? Or to be more specific, I think it is the only one we wouldn't want to propagate up, since the others would be raised by someone besides us I think,